CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+0.2.1
```

Comment:
You will need to update this one too https://github.com/acsylla/acsylla/blob/master/acsylla/version.py#L1